### DISCUSSION TITLE
Question about status refresh

### DISCUSSION THREAD

**Discussion by grahamck** [Hacksore/bluelinky#236] [General]
I believe that i understand that a status command can query either the current cached status on the server, or can directly query the car. I have incorporated this into my home automation system (homeseer) to alert me when things that are important to me  change. 

I have read that bluelink caps at 200 queries a day, is that still a known quantity? 

Directly querying the car sure seems to work, but i wonder if it is overkill. In particular, i am wondering when the cached status on the server is updated? For instance, If the car is unlocked, does it alert the server? If it did then i could query the server more often to look for changes instead of refreshing and going directly to the car every time.

Thanks!

**Comment by Hacksore** [comment]
Not exactly sure how the server cache is updated, maybe someone else can comment.

As for limits we do have this table which might still be accurate.
https://github.com/Hacksore/bluelinky/wiki/API-Rate-Limits

**Reply by grahamck** [reply]
Thanks!

 Ouch - 30 times a day is a pretty low limit!

**Comment by grahamck** [comment]
Did some testing. Doing a refresh I hit a limit at 50.  That limit does not appear to count if you are reading server cache with refresh set to false. 

Locked the car from the app, the server cache updated and I was able to see the change. 

Unlocked it, the server cache was again updated and I was able to read the change.

**Reply by moses19850** [reply]
Hi, how did you set up to recognize a change?
Thanks
